frontend/src/components/App.sass

```diff
@@ -0,0 +1,40 @@
+@charset "utf-8";
```

Comment:
not sure but I didn't know sass let the use of semicolons.
it runs fine but it show up red here.